showcase/src/app/news/routes.ts

```diff
@@ -0,0 +1,11 @@
+import { Ng2StateDeclaration } from "@uirouter/angular";
+import { NewsComponent } from "./news-component";
+
+export const NEWS_STATES: Ng2StateDeclaration[] = [
+	{
+		name: "news",
+		url: "^/news",
```

Comment:
Why declare `url: "^/news"` here and in demo module, declare  `url: "^/dropdown"` ?

Reply:
?? I don't think we have `url: "^/dropdown"` in the demo module... we do have `url: "/dropdown"``.

In any case, if you are wondering about the `^`, that is used to avoid double slash "//" in the URL after the domain (https://github.com/angular-ui/ui-router/wiki/URL-Routing#absolute-routes-). Otherwise, when we navigate to a demo page we will have a URL with double slashes (http://localhost:3000///demo/somePage") 😕